app/models/zip_code_search.rb

```diff
@@ -0,0 +1,29 @@
+class ZipCodeSearch
+  def initialize(params)
+    @params = params
+  end
+
+  def error_message
+    'There was a problem locating that ZIP Code. Mind trying your search again?'
+  end
+
+  def result
+    zip_code_tabulation_area = ZipCodeTabulationArea.find_by(zip_code: search_params[:postal_code])
```

Comment:
Does this + other searches set a maximum radius? It appears to just take the X closest, which in some places (intl) can probably include a huge geographical area.

Reply:
Right, there's currently no bounds being set, so a user could conceivably click through all 30-plus pages of results and see completely useless results. The designs in Zeplin have an optional "show me results within `x` miles" dropdown that we haven't yet implemented.

Reply:
We have also discussed limiting results to the same country as the origin, reasoning that going through customs or a border crossing to visit a TO is unworkable, but then again, maybe that actually happens in the EU?